src/vs/workbench/contrib/notebook/browser/controller/cellOutputActions.ts

```diff
@@ -104,3 +105,45 @@ function getOutputViewModelFromId(outputId: string, notebookEditor: INotebookEdi
 
 	return undefined;
 }
+
+export const OPEN_OUTPUT_COMMAND_ID = 'notebook.cellOutput.openInTextEditor';
+
+registerAction2(class CopyCellOutputAction extends Action2 {
```

Comment:
should be a different class name (or just remove the name)

Reply:
good catch!